LayoutTests/inspector/console/js-completions-expected.txt

```diff
@@ -13,3 +13,5 @@ PASS: Completions should still contain myVariable before cache is cleared.
 PASS: Completions should change after cache is cleared.
 PASS: Completions should not contain myVariable after it's deleted by code run in console.
 
+-- Running test case: console.jsCompletions.completionOrdering
+
```

Comment:
we should at least have _something_ be output from the test as otherwise if this becomes flaky it's much harder to understand where in the tests things are going wrong

i.e. i'd err on the side of over-logging (even if it's repetitive) as that there's at least some record of what's going on